src/components/elements/Ribbon.js

```diff
@@ -3,16 +3,17 @@ import { FONT_FAMILY } from '../../config/styles'
 
 export const Ribbon = `
   ${FONT_FAMILY}
-  font-size: 0.9rem;
+  font-size: 1rem;
   border-radius: 2px;
-  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.45), 0 0 2px 0 rgba(0, 0, 0, 0.12);
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.25), 0 0 2px 0 rgba(0, 0, 0, 0.12);
   background-color: #ffffff;
-  border: solid 1px #002938;
+  border: solid 3px #be6045;
```

Comment:
@shakersinthedark is there any color variable for it in https://github.com/reactjsacademy/reactjsacademy/blob/master/src/config/styles.js ? if not, can you please create it